libethash-cl/kernels/cl/ethash.cl

```diff
@@ -259,8 +259,10 @@ __kernel void search(
     uint isolate
 )
 {
+#ifndef NO_FAST_EXIT
```

Comment:
It should be `#ifdef FAST_EXIT` or better `#if FAST_EXIT`.

Reply:
Done